e2e/app/component/select-menu.ts

```diff
@@ -1,12 +1,21 @@
-import {Page} from 'puppeteer';
+import Container from 'app/container';
 import BaseElement from 'app/element/base-element';
+import {xPathOptionToXpath} from 'app/element/xpath-defaults';
+import {ElementType, XPathOptions} from 'app/xpath-options';
+import {Page} from 'puppeteer';
 
-export default class SelectMenu {
+export default class SelectMenu extends Container {
+
+  static async forLabel(page: Page, xOpt: XPathOptions = {}, container?: Container): Promise<SelectMenu> {
```

Comment:
why is this called `forLabel`? Should it perhaps be called `waitForLabel`?

Reply:
For consistency. Element wrapper classes (Button, Link, etc.) have `forLabel` static function.

Reply:
It's confusing, because there's no label in the signature or in the body of the function.

Also, can you declare it in the base class and use `override`?

Reply:
My expectation is that a function name tells me what a function does. This doesn't have a verb in it. Perhaps this should be called `getSelectMenu`, and the function getting the wrapped element in radio buttons should be `getRadioButton`, and so on.